prow/ANNOUNCEMENTS.md

```diff
@@ -3,7 +3,11 @@
 ## New features
 
 New features added to each component:
- - *May 31, 2020* '--gcs-no-auth' in Deck is deprecated and not used anymore. We always
+ - *June 1, 2020* Using `job_url_prefix_config` with `gcs/` prefix is now deprecated. 
+    Please use a job url prefix without a storage provider suffix. From now on the storage
+    provider is appended automatically so multiple storage providers can be used for builds of 
+    the same repository. Handling of the old format will be removed in *September 2020*.
```

Comment:
No, this is way too short. Not sure if we can remove it all, because we need to keep it for at least as long as the retention period of logs for k8s/istio/openshift, because removing this will break old links (which doesn't mater if they are broken anyways). For us, the retention period of logs is three months. @fejta how long are logs kept for kube tests?

Reply:
Ah okay, then the grace period is fine but yeah, needs clarification :)